site/en/blog/web-sql-deprecation-timeline-updated/index.md

```diff
@@ -44,6 +44,6 @@ removal in insecure contexts and eventually all contexts is planned for Chromium
 - [🔮 In the future.] A
   [deprecation trial](/docs/web-platform/origin-trials/#deprecation-trials) to
   keep using Web SQL is available from **Chromium&nbsp;118**
```

Comment:
should this be `**Chromium&nbsp;117**` ?

Reply:
Oh, good catch. I'll open a quick follow-up PR.

Reply:
https://github.com/GoogleChrome/developer.chrome.com/pull/6726

Reply:
Thanks!

Am I correct in understanding that the "ChromeDate` for 117 is also when the deprecation trial registration will become available at https://developer.chrome.com/origintrials/#/trials/active or will that be earlier?

Reply:
That’s my understanding too.